docs/encyclopedia/worker-versioning-legacy.mdx

```diff
@@ -0,0 +1,265 @@
+---
```

Comment:
I left this out of sidebar.js and left off tags so it won't be indexed for search.  Double check that I didn't do anything stupid.